### DISCUSSION TITLE
SO Cases workflow

### DISCUSSION THREAD

**Discussion by peko19** [Security-Onion-Solutions/securityonion#7445] [Ideas] [answered]
Please would it be possible to:

1. Have a function to merge the existing Cases?
2. In the "escalate alert" context menu just under “+ Escalate to a new case” specificaly in the list “Attach event to a recently viewed case”  have all recently created Cases, not only these already viewed?
3.  When escalating a Case from multiple grouped alerts to somehow fill in more information to the Case? Custom search query and a time interval at least?

thanks

peko

**Comment by dougburks** [comment, marked as answer]
We will review these as time allows.
